src/main/java/jnr/constants/platform/darwin/Fcntl.java

```diff
@@ -17,8 +17,8 @@
 F_SETSIZE(43L),
 F_RDADVISE(44L),
 F_RDAHEAD(45L),
-F_READBOOTSTRAP(46L),
```

Comment:
Did these really go away on darwin?  Maybe someone added those incorrectly by hand?

Reply:
That's my guess...we have not tried to mechanically generate these things in years, probably because of the bugs I tried to fix in #41.

Reply:
It does appear it went away.

```
[] ~/projects/jnr-constants $ grep -r F_READBOOTSTRAP /usr/include
/usr/include/bsm/audit_fcntl.h:#define	BSM_F_READBOOTSTRAP	346	/* Darwin-specific. */
/usr/include/sys/fcntl.h: * 46,47 used to be F_READBOOTSTRAP and F_WRITEBOOTSTRAP 
/usr/include/sys/fcntl.h:/* fbootstraptransfer_t used by F_READBOOTSTRAP and F_WRITEBOOTSTRAP commands */
```